doc/install/index.rst

```diff
@@ -3,12 +3,12 @@
 Get Arbor
 =========
 
-Currently we offer three ways to get Arbor.
+Currently we offer two ways to get Arbor.
```

Comment:
I suppose Spack will be added shortly.

Reply:
Yes, but till then it's better to keep it correct :)